app/helpers/mod-if.js

```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
+
+export function modIf([value, modBy]) {
+    return (value + 1 % modBy === 0);
```

Comment:
Oops! Those order of operations. When it is actually working, it does make the columns look a little better when the page gets smaller though.